chainerx_cc/chainerx/cuda/memory_pool_test.cc

```diff
@@ -356,6 +357,26 @@ TEST(MemoryPoolTest, FreeUnusedBlocksSplitAndFreeHead) {
     EXPECT_EQ(allocator->free_called(), 0);
 }
 
+TEST(MemoryPoolTest, FreeUnusedBlocksThreadSafe) {
+    static constexpr size_t kRepeat = 100U;
+    MemoryPool memory_pool{0, std::make_unique<FixedCapacityDummyAllocator>(0xffffffffU)};
+
+    testing::RunThreads(2, [&memory_pool](size_t thread_index) {
+        for (size_t i = 0; i < kRepeat; ++i) {
+            switch (thread_index) {
+                case 0: {
+                    void* ptr = memory_pool.Malloc(1U);
+                    memory_pool.Free(ptr);
+                    break;
+                }
+                case 1:
```

Comment:
Enclose this section in a brace.

Reply:
There's no variable declaration here. Thus no need of braces.